@W-16470063 - Update to-publish-assets-maven.adoc (#284)
mulesoft/docs-exchange: modules/ROOT/pages/to-publish-assets-maven.adoc

</build>
----

> == Publish a parent pom project and a parent pom dependant project to Exchange

Reviewer: ```suggestion
== Publish a Parent POM Project and a Parent POM Dependant Project to Exchange
```

> Parent poms are published to Exchange as custom assets. See previous section in order to publish a custom asset.

Reviewer: ```suggestion
Parent POMs are published to Exchange as custom assets. See the previous section to publish a custom asset.
```

> If you want to publish a maven project, like a mule application, using the Mule Maven Plugin, and your maven project depends on a Parent Pom that has already included the exchange-mule-maven-plugin, you need to https://maven.apache.org/guides/mini/guide-configuring-plugins.html#Using_the_.3Cinherited.3E_Tag_In_Build_Plugins[prevent the inheritance]:

Reviewer: ```suggestion
If you want to publish a Maven project, like a Mule application, using the Mule Maven plugin, and your Maven project depends on a parent that is already included in the exchange-mule-maven-plugin, you must https://maven.apache.org/guides/mini/guide-configuring-plugins.html#Using_the_.3Cinherited.3E_Tag_In_Build_Plugins[break the inheritance]:
```

Author: This is not why i actually mean. 
The desired output will be something like: "if your maven project uses the mule-maven-plugin, and depends on a parent on that already includes the exchange-mule-maven-plugin, you must break the inheritance"
